src/main/java/frc/robot/Subsystem/Climb/ClimbConstants.java

```diff
@@ -3,6 +3,9 @@
 public interface ClimbConstants {
     int M_CLIMBINGRIGHT_MOTOR_ID = 0; //TODO: find id
     int M_CLIMBINGLEFT_MOTOR_ID = 0; //TODO: find id
```

Comment:
add `  _  ` between CLIMBING to RIGHT and LEFT

Reply:
V